Trigger/Algo/include/Trigger/Event/TrigMip.h

```diff
@@ -0,0 +1,54 @@
+#ifndef TRIGGER_EVENT_TRIGMIP_H
+#define TRIGGER_EVENT_TRIGMIP_H
+
+#include "TObject.h"  //For ClassDef
+
+namespace trigger {
+
+// Forward declaration needed by typedef
+class TrigMip;
+typedef std::vector<TrigMip> TrigMipCollection;
```

Comment:
I don't know who started this trend, but it is not needed and (I would argue) more confusing than just using `std::vector<TrigMip>` in your producers/analyzers.

```suggestion
```

This more of an opinion though so you can ignore it if need be.